src/callable/operators.rs

```diff
@@ -307,6 +313,13 @@ impl Callable for InfixColon {
         } else {
             let start: i32 = stack.eval(arg1)?.as_integer()?.try_into()?;
             let end: i32 = stack.eval(arg2)?.as_integer()?.try_into()?;
+            if start > end {
+                return Error::Other(
+                    "Colon operator requires end >= start, use a tertiary call instead (x:y:z)."
+                        .to_string(),
+                )
```

Comment:
This is super pedantic, but I would like to avoid naming operators by their symbols. Instead of saying "Colon operator" (because the _symbol_ is a colon, but the _operator_ is for making ranges), describe the functionality:

>  "Ranges with a `by` should have an end >= start. Provide a negative `by` for the range using `start:by:end`"

I would also suggest introducing this as a new error variant (I'm also very guilty of shoehorning errors into the "Other", but I'm trying to clean this up throughout the codebase).